src/flexalloc_mm.c

```diff
@@ -1396,10 +1395,15 @@ fla_object_read(const struct flexalloc * fs, struct fla_pool const * pool_handle
     err = fla_xne_sync_seq_r_nbytes(fs->dev.dev, r_soffset, r_len, buf);
   else
   {
+    struct fla_strp_params sp;
     sp.strp_nobjs = pool_entry->strp_nobjs;
-    sp.strp_nbytes = pool_entry->strp_nbytes;
-    sp.obj_nlbs = pool_entry->obj_nlb;
-    err = fla_xne_sync_strp_seq_x(fs->dev.dev, r_soffset, r_len, buf, &sp, false);
+    sp.strp_chunk_nbytes = pool_entry->strp_nbytes;
+    sp.faobj_nlbs = pool_entry->obj_nlb;
+    sp.xfer_snbytes = r_offset;
+    sp.xfer_nbytes = r_len;
+    sp.strp_tnbytes = (uint64_t)pool_entry->strp_nobjs * pool_entry->obj_nlb * fs->geo.lb_nbytes;
```

Comment:
I was thinking we could rename this to reflect that is the total number of bytes of the striped object, maybe strp_obj_tnbytes ?